query/src/main/scala/filodb/query/exec/ExecPlan.scala

```diff
@@ -198,6 +198,16 @@ trait ExecPlan extends QueryCommand {
     ((transf :+ curNode) ++ childr).mkString(if (useNewline) "\n" else " @@@ ")
   }
 
+  final def getPlan(level: Int = 0): Seq[String] = {
+    val transf = rangeVectorTransformers.reverse.zipWithIndex.map { case (t, i) =>
+      s"${"-"*(level + i)}T~${t.getClass.getSimpleName}(${t.args})"
```

Comment:
Might want to put this part which is shared in a function

Reply:
Missed this before. Added function to print RangeVectorTransformers